### PR TITLE
adding LABELS column to table

### PR DESCRIPTION
**Pull request by 100starnight**
tableLabel

### CODE REVIEW

File: packages/directory/addon/components/dir-item-label-cell.ts [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+interface DirItemNameCellComponentArgs {
+  value: TODO<{ constructor: { modelName: string }; modelId: string }>;
+}
+
+export default class DirItemLabelCellComponent extends Component<DirItemNameCellComponentArgs> {
+  /**
+   * @property {String} itemId - the id of the model or the tempId of an unsaved model
+   */
+  get itemId() {
```
Comment (100starnight) [inline packages/directory/addon/components/dir-item-label-cell.ts:22]: This isn't actually used anywhere, but I get errors if I have an empty component so I left it in for now

**Comment by jkusa** [comment]
@100starnight can you please post a screenshot?

File: packages/directory/addon/components/dir-item-label-cell.ts [hunk 2]
```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright 2021, Yahoo Holdings Inc.
+ * Licensed under the terms of the MIT license. See accompanying LICENSE.md file for terms.
+ *
+ * This component won't be used directly. It is passed to ember-light-table as a custom cell component.
+ * Ember-light-table will pass any parameters in through the value attribute.
+ *
+ * <DirItemNameCell
+ *  @value={{@item}}
+ * />
```
Comment (kevinhinterlong) [inline packages/directory/addon/components/dir-item-label-cell.ts:10]:
```suggestion
```

File: packages/directory/addon/components/dir-item-label-cell.hbs [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{{!-- Copyright 2021, Yahoo Inc. --}}
```
Comment (kevinhinterlong) [inline packages/directory/addon/components/dir-item-label-cell.hbs:1]: mit license

File: packages/directory/addon/components/dir-item-label-cell.hbs [hunk 4]
```diff
@@ -0,0 +1,9 @@
...
+{{#each this.labels as |label|}}
+<DenaliChip
```
Comment (kevinhinterlong) [inline packages/directory/addon/components/dir-item-label-cell.hbs:3]: indent

File: packages/directory/addon/components/dir-item-label-cell.ts [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+interface DirItemLabelCellComponentArgs {
+  value: ReportModel | DashboardModel;
+}
+
+export default class DirItemLabelCellComponent extends Component<DirItemLabelCellComponentArgs> {
+  /**
+   * @property {String[]} labels - the labels that will be applied to this row of the table
+   */
+  get labels() {
+    let labs: string[] = [];
```
Comment (kevinhinterlong) [inline packages/directory/addon/components/dir-item-label-cell.ts:25]: `labs` -> `labels`

**Review by kevinhinterlong — COMMENTED** [review]
small stuff

**Review by kevinhinterlong — APPROVED** [review]

**Review by cythrawll — APPROVED** [review]

**Pull request merged**